File: docs/components_page/components/nav/link_based.R
library(dashBootstrapComponents)

nav_link_based <- dbcNav(
  list(
    dbcNavLink("Active", active = TRUE, href = "#"),
    dbcNavLink("A link", href = "#"),
    dbcNavLink("Another link", href = "#"),
    dbcNavLink("Disabled", disabled = TRUE, href = "#")
  )
)

File: docs/components_page/components/nav/link_based.jl
using DashBootstrapComponents

nav = dbc_nav([
    dbc_navlink("Active", active = true, href = "#"),
    dbc_navlink("A link", href = "#"),
    dbc_navlink("Another link", href = "#"),
    dbc_navlink("Disabled", disabled = true, href = "#"),
]);
